gensim/api/__init__.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import print_function
+from __future__ import absolute_import
+import json
+import os
+import tarfile
+import logging
+import sys
+
+try:
+    import urllib.request as urllib
+except ImportError:
+    import urllib
+
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib2 import urlopen
+
+logger = logging.getLogger('gensim.api')
+
+
+def download(file_name):
+    url = "https://github.com/chaitaliSaini/Corpus_and_models/releases/"
+    url = url + "download/" + file_name + "/" + file_name + ".tar.gz"
+    user_dir = os.path.expanduser('~')
+    base_dir = os.path.join(user_dir, 'gensim-data')
+    extracted_folder_dir = os.path.join(base_dir, file_name)
+    if not os.path.exists(base_dir):
+        logger.info("Creating {}".format(base_dir))
```

Comment:
Please use logging correct formatting `logger.info("Creating %s", base_dir)` (here and anywhere). Look at [this example](https://docs.python.org/2/howto/logging.html#logging-variable-data).